.github/ci_setup.sh

```diff
@@ -15,11 +15,14 @@ PLATFORM_NAME=$1
 
 # Set up the correct environment for the SYCL tests.
 if [ "${PLATFORM_NAME}" = "SYCL" ]; then
-   source /opt/intel/oneapi/setvars.sh --include-intel-llvm
+   if [ -f "/opt/intel/oneapi/setvars.sh" ]; then
+      source /opt/intel/oneapi/setvars.sh --include-intel-llvm
+   fi
```

Comment:
Curious. This is not a bad addition per se, but does one of our current oneAPI images not have oneAPI installed under `/opt/intel/oneapi`? 😕

But sure, we may very well want to add SYCL builds not based on oneAPI at one point. 🤔